### DISCUSSION TITLE
科研通 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#194] [模板讨论]
科研通 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
2024-02-06 17:13:09
日志：Failed at 2/4 request,
Fail assert: {"re": "\\\\u767b\\\\u5f55\\\\u6210\\\\u529f", "from": "content"} from success_asserts,
Request URL: https://www.ablesci.com/site/login

**Reply by wjf0214** [reply]
你都不是最新的吧。最新是5条请求。你这显示4条。
